packages/blade/src/components/Button/BaseButton/buttonTokens.ts

```diff
@@ -0,0 +1,36 @@
+const typography = {
+  onDesktop: {
+    fonts: {
+      size: {
+        large: 200,
```

Comment:
wondering if we shall use the key from the global token here? so that we never miss this. Although TS will fail if this messes up regardless but just to keep things in sync? thoughts? I just realized that even in typography I've missed this

Reply:
Check this commit. See if this makes a bit more sense [`368d21f` (#487)](https://github.com/razorpay/blade/pull/487/commits/368d21f4b574230b99dfdb82cfc6fc178cac8d02)